assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,164 @@
+// <----------------------------- ARROW FUNCTION ----------------------------->
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+let anotherFunction = (param) => param;
 
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
+let add = (param1, param2) => param1 + param2;
 
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
+let subtract = (param1, param2) => param1 - param2;
 
 // exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+const triple = array => {
```

Comment:
I was gonna comment that you didn't need parenthesis for single parameter, but just saw this part and had to tell you this.